.github/workflows/test-daily.yaml

```diff
@@ -0,0 +1,30 @@
+name: Tests (DAILY)
```

Comment:
@rafaeldtinoco @grantseltzer I think the name of the badge as "Tests (DAILY)" isn't so crystal since this workflow only wants to compile against upstream libbfpgo. How about `libbpfgo@main (DAILY)`?

Reply:
Just ignore the prior question, it's better to keep a generic naming for that, so we can insert new specific jobs if necessary.